<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>601</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>801</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="playerOne">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>67</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Player 1&lt;/p&gt;&lt;p&gt;Money:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerTwo">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>90</y>
     <width>67</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Player 2&lt;/p&gt;&lt;p&gt;Money:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerThree">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>160</y>
     <width>67</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Player 3&lt;/p&gt;&lt;p&gt;Money:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerFour">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>230</y>
     <width>67</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Player 4&lt;/p&gt;&lt;p&gt;Money:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="plOneMoney">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>49</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1500</string>
   </property>
  </widget>
  <widget class="QLabel" name="plTwoMoney">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>120</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1500</string>
   </property>
  </widget>
  <widget class="QLabel" name="plThreeMoney">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>190</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1500</string>
   </property>
  </widget>
  <widget class="QLabel" name="plFourMoney">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>260</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removePl">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Player</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addMoneyButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>530</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Money</string>
   </property>
  </widget>
  <widget class="QComboBox" name="playerAddMoney">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>500</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Player One</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Player Two</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="addMoneyAmount">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>500</y>
     <width>52</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addPlayer">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>420</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Player</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
